src/v/config/node_config.cc

```diff
@@ -169,6 +172,46 @@ node_config::node_config() noexcept
       "injection",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , debug_bundle_write_dir(
+      *this,
+      "debug_bundle_write_dir",
+      "The full path to the directory where debug bundle zip files are stored "
+      "(default: "
+      "/tmp)",
+      {.visibility = visibility::user},
+      "/tmp",
```

Comment:
should the default be a directory that is relative to the configured data directory?

Reply:
From the other thread https://github.com/redpanda-data/redpanda/pull/10488#discussion_r1198870474

I agree that the data dir is a reasonable place. I also think that we can remove this config option and simply always write to the data dir since it will always be there.